### PR TITLE
rescue on some of the bad monitoring IDs

### PR DESCRIPTION
**Pull request by jread-usgs**
See #184. 

This creates additional tasks for organizations that have bad monitoring IDs and adds those pulls to the task table.

### CODE REVIEW

**Comment by jread-usgs** [comment]
@limnoliver small extension of what you already reviewed

File: 6_wqp_fetch/src/fetch_wqp_data.R [hunk 1]
```diff
@@ -136,21 +134,24 @@ pull_wqp_data <- function(data_file, wqp_sites, characteristicName, dummy){
...
+  } else {
+    wqp_args$siteid <- pull(wqp_sites, MonitoringLocationIdentifier)
   }
 
 
   # do the data pull
   # first pull using readWQPdata, then if that fails, try POST
   wqp_dat <- suppressMessages(wqp_POST(wqp_args))
 
+  if (wqp_sites$pull_org){
```
Comment (limnoliver) [inline 6_wqp_fetch/src/fetch_wqp_data.R:152]: Do you need a subset here, as above (`wqp_sites$pull_org[1]`)?

File: 6_wqp_fetch/src/fetch_wqp_data.R [hunk 2]
```diff
@@ -136,21 +134,24 @@ pull_wqp_data <- function(data_file, wqp_sites, characteristicName, dummy){
...
+    wqp_dat <- filter(wqp_dat, MonitoringLocationIdentifier %in% c(wqp_sites$MonitoringLocationIdentifier))
```
Comment (limnoliver) [inline 6_wqp_fetch/src/fetch_wqp_data.R:153]: small note - the c() is not needed here?

**Review by limnoliver — APPROVED** [review]
Looks good, Jordan -- glad you were able to save those sites!

File: 6_wqp_fetch/src/fetch_wqp_data.R (hunk 2 above)
Comment (jread-usgs) [reply 6_wqp_fetch/src/fetch_wqp_data.R:153]: correct. Will fix

File: 6_wqp_fetch/src/fetch_wqp_data.R (hunk 1 above)
Comment (jread-usgs) [reply 6_wqp_fetch/src/fetch_wqp_data.R:152]: yes, this one needs that (same as above), but works like this just with an warning that it is using the first of the vector. Will update this one too.

**Pull request merged**